refinery/templates/registration/registration_form.html

```diff
@@ -114,6 +114,7 @@ <h4>Terms of Service</h4>
       </p>
 	</form>
 </div>
+</div>
```

Comment:
The code above should be indented accordingly. This will also help identifying missing tags later.